### ISSUE TITLE
SharpCap Error - msg.exe Doesn't Exist on Windows 11 Home

### ISSUE THREAD

**Issue by scottsousa** [linuxkidd/SharpCap-Eclipse-Sequencer#3]
Hi @linuxkidd ,

I ran into an issue today on one of my devices with Windows 11 Home. It turns out that `msg.exe` doesn't exist on this version. SharpCap stops execution and returns an error when testing the [latest version of your script](https://github.com/linuxkidd/SharpCap-Eclipse-Sequencer/commit/bb301f75b7eac0dbe4ba8714d2634dc9a8c956fd).

Here is a snippet of the log from the SharpCap Sequence Editor:

> 6:31:07 PM Starting  : Run 'C:\\Windows\\System32\\msg.exe' with parameters '* /time:30 Remove the Solar Filter!' and leave it running
> 6:31:07 PM Error     : 'An error occurred trying to start process 'C:\\Windows\\System32\\msg.exe' with working directory 'C:\Program Files\SharpCap 4.1 (64 bit)'. The system cannot find the file specified.' while running step Run 'C:\\Windows\\System32\\msg.exe' with parameters '* /time:30 Remove the Solar Filter!' and leave it running
> 6:31:07 PM End       : Sequence halted due to error.

I think there's a chance that `cmd /C "IF EXIST C:\\Windows\\System32\\msg.exe C:\\Windows\\System32\\msg.exe * /time:30 Remove the Solar Filter!"` could work to prevent errors on Windows 11 Home devices. What do you think?

**Comment by linuxkidd** [comment]
Dang... thanks for catching this, @scottsousa .. Microsoft allows users to download the executable from the web, but then I'd have to tell them how to copy it to their system32 folder.  While easy, I don't want to expect users to do that.

Could you test the following on Windows Home?  I always upgrade my systems to Pro, so I'm not sure if this is also limited on Home...
```
powershell -c "(New-Object -ComObject Wscript.Shell).Popup('Urgent! --- Apply your solar filters! --- Urgent!',2,'Safety!')"
```
Note this should pop up, then disappear after 2 seconds if 'OK' isn't clicked.

I do think this is sub-optimal since it will pop open a PS window until the dialog closes .. and i'm open to other options :)

Thanks!
LK

**Comment by linuxkidd** [comment]
Closed by #5
